oauth2server/proj/settings/docker.py

```diff
@@ -0,0 +1,35 @@
+from proj.settings.default import *
+
+
+# Standard instance of a logger with __name__. We are using this so that our root folder has our logger defined.
+stdlogger = logging.getLogger(__name__)
+stdlogger.info("*** Local settings are being used. ***")
```

Comment:
Maybe replace this with 'Docker settings being used'
If there's any point in logging this at all. If we're going to log it out it would probably make more sense to do it where the setting are called rather than in this file?

Reply:
Changed logging to say 'docker' instead of 'local'